extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -166,3 +166,61 @@
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+/** Inline attachment specifications */
```

Comment:
This comment can be more direct, like 
`For amp-story-inline-page-attachment-v2 experiment.`